airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -59,7 +59,55 @@
 FARGATE_FULL_NAME = "AWS Fargate profiles"
 
 
-class EksCreateClusterOperator(BaseOperator):
+class BaseEksCreateOperator(BaseOperator):
+    """
+    Base class for EKS Operators that create cluster resources. This includes
+    EksCreateClusterOperator, EksCreateNodeGroupOperator and EksCreateFargateProfileOperator.
+
+    """
```

Comment:
```suggestion
    """Base class for EKS Operators that create cluster resources.

    This includes EksCreateClusterOperator, EksCreateNodeGroupOperator and
    EksCreateFargateProfileOperator.
    """
```

I’m not sure we should have the second line though. The list of classes doesn’t really add much useful information for maintenance, and is very easy to get stale.

Reply:
Also maybe a `:meta private:` would be a good idea? I don’t think we expect users to call this class directly. (Or rename this class to `_BaseEksCreateOperator`)